packages/components/psammead-idt1-include/src/fakeinclude.html

```diff
@@ -0,0 +1,11 @@
+<div id="placeholder">Uninitialised include</div>
+<script>
+  define('initialiser', function() {
+    var element = document.getElementById('placeholder');
+    element.innerHTML = 'Initialised include';
+  });
+
+  require(['initialiser'], function(initialiser) {
+    initialiser();
+  });
+</script>
```

Comment:
This is meant to be a minimal replication of a typical include, it:
- Defines a div with the id `placeholder` with some text indicating the include has not been initialised
- It defines a module to initialise the include using require
  -  The code get the placeholder element and changes its text to indicate it is initialised
- `require` is called to resolve the module called `initialiser` and execute it, replacing the text in the div to be "Initialised include"

This tests that require can be used via it's two entry point methods, `define` and `require`; emulating a typical pattern found in idt1 includes.